diff --git a/css/mystyles.css b/css/mystyles.css
deleted file mode 100644
index 60f51bf..0000000
--- a/css/mystyles.css
+++ /dev/null
@@ -1 +0,0 @@
-/* This is a default css file. */
diff --git a/js/myjs.js b/js/myjs.js
deleted file mode 100644
index 8d891a1..0000000
--- a/js/myjs.js
+++ /dev/null
@@ -1 +0,0 @@
-// This is a default js file.
diff --git a/tttest.txt b/tttest.txt
new file mode 100644
index 0000000..00fbf27
--- /dev/null
+++ b/tttest.txt
@@ -0,0 +1 @@
+this is tttest txt.
